two point sources

```diff
@@ -0,0 +1,102 @@
+#%%
+"""
+
+Two dipole model.
+Consider the two point sources are at (0, y, 0) and (0, -y, 0).
+Small distance comparing to target position (X_0, Y_0)
+
+"""
+
+#y = np.linspace(-5, 5, 50)
+#z = np.linspace(-0.1, 0.1, 4)
+t = np.linspace(0, 1e-8, 501)
+
+def V_y(omega, t):
+    return np.exp(1j*omega*t)
+
+"""
+Voltage field of the point source at Y = y. 
+Providing electric field E_y at the target (X_0, Y_0) or (R, Theta).
+Amplitude normalized.
+"""
+
+def V_ny(omega, t, y, theta, c):
```

Comment:
define fuction, add the descrption to this function, and add which tyoe and which inputs you are using, and whta it returns